src/Microsoft.DotNet.Tools.Publish/PublishCommand.cs

```diff
@@ -236,5 +186,58 @@ private static void PublishFiles(IEnumerable<LibraryAsset> files, string outputP
                 File.Copy(file.ResolvedPath, Path.Combine(outputPath, Path.GetFileName(file.ResolvedPath)), overwrite: true);
             }
         }
+
+        public static int Main(string[] args)
```

Comment:
I will put it into separate class although I will not change it everywhere yet. Let's make it bake a while in here and see if design is viable first and if it proves itself good then we can start creating massive merge conflicts :smiley: